theano/tensor/blas_headers.py

```diff
@@ -834,7 +833,7 @@ def blas_header_text():
 
     /* Single Precision */
 
-        void sgemv_(char*, const int*, const int*, const float *, const float *, const int*, const float *, const int*, const float *, float *, const int*);
+        void sgemv_(char*, const int*, const int*, const float *, %(const)s float *, const int*, %(const)s float *, const int*, const float *, float *, const int*);
```

Comment:
Why this change?

Reply:
Because I have to wrap A and x to PyArrayObjects, but NumPy functions don't take constant data pointers.

Reply:
ok, it is the same as for gemm.